packages/apollo-voyager-conflicts/src/handleConflict.ts

```diff
@@ -0,0 +1,49 @@
+import { SyncServerError } from './detectConflict'
+
+function RETURN_TO_CLIENT (conflict: SyncServerError, currentRecord: ConflictResolutionData, client: ConflictResolutionData): SyncServerError {
+  console.warn(`Conflict detected. Server: ${currentRecord} client: ${client}`)
+  return conflict
+}
+
+function CLIENT_WINS (conflict: SyncServerError, currentRecord: ConflictResolutionData, client: ConflictResolutionData): ConflictResolutionHandler {
+  console.warn(`Conflict detected. Server: ${currentRecord} client: ${client}`)
+  const newVersion = currentRecord.version + 1
+  const newRecord = { ...client, version: newVersion }
+  return newRecord
+}
+
+function SERVER_WINS (conflict: SyncServerError, currentRecord: ConflictResolutionData, client: ConflictResolutionData): SyncServerError {
+  console.warn(`Conflict detected. Server: ${currentRecord} client: ${client}`)
```

Comment:
-1000 on using console.. I believe that we settled with debug package so we can required it here.